docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker-in-Docker ("dind") in a kata container.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
+
+## Background
+
+Docker in Docker ("DinD") is the colloquial name for the ability to run `docker` from inside a container (whether provided by `docker` or otherwise) pattern for running `docker` from *inside* an existing `docker` container. You can read more about Docker-in-docker at the following links:
+
+- [The original announcement of DinD](https://www.docker.com/blog/docker-can-now-run-within-docker/)
+- [`docker` image DockerHub page](https://hub.docker.com/_/docker/) (this page lists the `-dind` releases)
+
+**NOTE** While in the past kata containers did not contain the [`overlay` kernel module (aka OverlayFS)][overlayfs], it *now does* with the merging of the following pull requests:
+
+- [PR #1384](https://github.com/kata-containers/kata-containers/pull/1384)
+- [PR #716](https://github.com/kata-containers/packaging/pull/716)).
```

Comment:
It would be better to remove these links and state that Kata version x.y.z supports it.

Reply:
- [ ] Remove links to PRs, replace with version of Kata in which the PRs landed